geometry_msgs/Pose[] path
---
string resp

